example_test.go

```diff
@@ -1,9 +1,10 @@
 package quantile_test
 
 import (
-	quantile "." // import fully qualified in your code
 	"fmt"
 	"time"
+
+	"github.com/koalacxr/quantile"
```

Comment:
Alternatively, could this test file just be in the `quantile` package, and the `quantile.`s below removed?